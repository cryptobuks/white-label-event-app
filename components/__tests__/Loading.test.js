import React from 'react';
import renderer from 'react-test-renderer';
import { Loading } from '../';

describe('Loading', () => {
  it('renders correctly', () => {
    const comp = <Loading />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
