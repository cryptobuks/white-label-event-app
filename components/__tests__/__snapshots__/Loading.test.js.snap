// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="black"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
